src/footer.stories.mdx

```diff
@@ -0,0 +1,144 @@
+import { Meta, Canvas, Story } from '@storybook/addon-docs/blocks';
+import { Footer } from './Footer';
+import { Divider, DividerOrientation } from './Divider';
+import { Link } from './Link';
+import { Stack } from './Stack';
+import { Grid, GridItem } from './Grid';
+import { FooterPartner } from './FooterPartner';
+import { FooterList, FooterListItem } from './FooterList';
+import { StoreMenu } from './StoreMenu';
+import styled from 'styled-components';
+import { fontFamily, fontWeight, space, color, borderRadius } from '@core-ds/primitives';
+import stores from './stores.json';
+import languages from './languages.json';
```

Comment:
This is unused. What happened to the language dropdown? Are we not going to include that in this initial version?

Reply:
As @sterlinghirsh said on this [Slack conversation](https://inkofpixel.slack.com/archives/C01K32B4FPX/p1611872439031500?thread_ts=1611858325.013200&cid=C01K32B4FPX) Shopify stores are single language, therefore the language selector is not needed, unless I'm missing something 🤔

Reply:
That is true, but we are planning to use this footer on both Shopify and ifixit.com. ifixit.com supports many languages.